Marlin/src/pins/pins_TRIGORILLA_14.h

```diff
@@ -31,22 +31,24 @@
 #define IS_RAMPS_EFB
 
 // FAN0 / D9  - Typically used for the part fan on Anycubic Delta devices
-#define FAN_PIN 9
+#define FAN_PIN               9
 
 // FAN1 / D7  - Typically unused, can be allocated as Case Fan
+#define FAN1_PIN              7
 
 // FAN2 / D44 - Typical Extruder Fan on Anycubic Delta devices
 #define FAN2_PIN              44
 #define ORIG_E0_AUTO_FAN_PIN  44
 
+#define HEATER_1_PIN          45
+
 #include "pins_RAMPS.h"
 
 // TODO 1.4 boards do have an E1 stepper driver. However the pin definitions
 // from pins_RAMPS.h are incorrect for this board. e.g., Pin 44 is the Extruder fan.
-#undef E1_STEP_PIN
-#undef E1_DIR_PIN
-#undef E1_ENABLE_PIN
-#undef E1_CS_PIN
+#if PIN_EXISTS(E1_CS) && PIN_EXISTS(FAN2) && E1_CS_PIN == FAN2_PIN
+  #warning "E1_CS_PIN and FAN2_PIN are set to the same pin."
+#endif
```

Comment:
Pins files define the pins that should be used, and don't specify that a pin actually _is_ used. So this warning will be unnecessary for the case where (1) no second fan is being used, (2) the second E plug is unused, etc. If this warning is needed, it would be best to have it in `SanityCheck.h` at a point where more conditions can be checked. But I'm not sure at the moment how we can determine that the `CS_PIN` and `FAN2_PIN` are definitely being used even there.